recipe/meta.yaml

```
@@ -58,6 +52,11 @@ outputs:
         - zlib
       run:
         - ucrt  # [win]
+        # While zlib and openssl should in theory be marked as a private library
+        # grpc.pc (the pkgconfig file) seems to require it at link time
+        # for any package linking to grpc
+        - zlib
+        - openssl
```

We will also need to revert the part that added abseil
```suggestion
        # While zlib and openssl should in theory be marked as a private library
        # grpc.pc (the pkgconfig file) seems to require it at link time for any
        # package linking to grpc; similarly for building against a static abseil,
        # which doesn't have a run-export but is needed at link-time.
        - libabseil-static ={{ libabseil_static }}=cxx11*  # [win]
        - openssl
        - zlib
```

---

this is required for windows? so strange. ok.

---

It's a similar issue as zlib. Even more so for compiling against a static build, having abseil around shouldn't be necessary for linking against grpc, but because it sets all the abseil libs as `PUBLIC` CMake insists on finding them.

This is something I'll have to fix more urgently than the unix side, because it blocks grpc from being co-installed with another abseil.